arangod/RestServer/FlushFeature.cpp

```diff
@@ -94,26 +94,18 @@ std::tuple<size_t, size_t, TRI_voc_tick_t> FlushFeature::releaseUnusedTicks() {
 
   {
     std::lock_guard lock{_flushSubscriptionsMutex};
-
-    // find min tick and remove stale subscriptions
-    for (auto itr = _flushSubscriptions.begin();
-         itr != _flushSubscriptions.end();) {
-      auto entry = itr->lock();
-
-      if (!entry) {
-        // remove stale
-        itr = _flushSubscriptions.erase(itr);
-        ++stale;
-      } else {
-        LOG_TOPIC("5a4fb", TRACE, arangodb::Logger::FLUSH)
-            << "found flush subscription: " << entry->name() << ", tick "
-            << entry->tick();
-
```

Comment:
The PR removes this log message, but it shouldn't.
This log message helped us diagnose and prove that there are problems with stale links in the first place!

Reply:
Yes, it's not intentional, will be fixed